activejob/lib/active_job/test_helper.rb

```diff
@@ -10,6 +10,7 @@ module TestHelper
 
     def before_setup # :nodoc:
       test_adapter = queue_adapter_for_test
+      self.class.to_s.gsub(/Test\z/, "").safe_constantize
```

Comment:
This line is confusing, as we constantize something, and don't assign it to any variable.
Would be nice to add a comment in it, to make it more clear.